arch/arm/core/aarch64/reset.S

```diff
@@ -15,6 +15,9 @@
 #include <arch/cpu.h>
 #include "vector_table.h"
 #include "macro_priv.inc"
+#ifdef CONFIG_STACK_SENTINEL
+#include <kernel_structs.h> // For STACK_SENTINEL definition
```

Comment:
hm, i think we 're not using '//' comment-style at all

Reply:
My apologies - I'm used to a style that does.

- [ ] Change `//`-style comments to `/*`-style.